autogpt_platform/backend/backend/blocks/generic_webhook/_webhook.py

```diff
@@ -17,7 +17,7 @@ class GenericWebhooksManager(ManualWebhookManagerBase):
 
     @classmethod
     async def validate_payload(
-        cls, webhook: Webhook, request: Request
+        cls, webhook: Webhook, request: Request, credentials: Credentials | None = None
```

Comment:
We had to change the signature to support being able to query the api when the webhook was triggered. This is currently unique to airflows methodology